native/dlascl.go

```diff
@@ -0,0 +1,72 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import (
+	"math"
+
+	"github.com/gonum/lapack"
+)
+
+// Dlascl multiplies a rectangular matrix by a scalar.
+func (impl Implementation) Dlascl(kind lapack.MatrixType, kl, ku int, cfrom, cto float64, m, n int, a []float64, lda int) {
```

Comment:
They are for the banded types that have not yet been implemented in this function. This is a stupid function, especially by LAPACK standards. It should be Dgescl or something like that rather than switch over all of the matrix types. The input parameters have a variety of meanings depending on the matrix type.